index.html

```diff
@@ -3,221 +3,282 @@
+          </li>
+          <li class="socical-item"><img class="socical-icon" src="./assets/icons/instagram-logo.svg"
+              alt="Instagram Icon"></li>
+          <li class="socical-item"><img class="socical-icon" src="./assets/icons/youtube-logo.svg" alt="Youtube Icon">
+          </li>
+        </ul>
+      </div>
+      <ul class="footer-columns-list">
+        <li class="footer-columns-item">
+          <ul class="columns-info-list">
+            <h5 class="columns-info-title">Shopping online</h5>
+            <li class="columns-info-item"><a class="columns-info-link" href="#">Order</a></li>
+            <li class="columns-info-item"><a class="columns-info-link" href="#">Status</a></li>
+            <li class="columns-info-item"><a class="columns-info-link" href="#">Shipping and Delivery</a></li>
+            <li class="columns-info-item"><a class="columns-info-link" href="#">Returns</a></li>
+            <li class="columns-info-item"><a class="columns-info-link" href="#">Payment</a></li>
+            <li class="columns-info-item"><a class="columns-info-link" href="#">Options</a></li>
+            <li class="columns-info-item"><a class="columns-info-link" href="#">Contact Us</a></li>
           </ul>
-          <ul class="footer-infomation">
-            <li>Information</li>
-            <li>Gift Cards</li>
-            <li>Find a store</li>
-            <li>Newsletter</li>
-            <li>Become a member</li>
-            <li> Site feedback</li>
+        </li>
+        <li class="footer-columns-item">
+          <ul class="columns-info-list">
+            <h5 class="columns-info-title">Information</h5>
+            <li class="columns-info-item"><a class="columns-info-link" href="#">Gift Cards</a></li>
+            <li class="columns-info-item"><a class="columns-info-link" href="#">Find a store</a></li>
+            <li class="columns-info-item"><a class="columns-info-link" href="#">Newsletter</a></li>
+            <li class="columns-info-item"><a class="columns-info-link" href="#">Become a member</a></li>
+            <li class="columns-info-item"><a class="columns-info-link" href="#">Site feedback</a></li>
           </ul>
-          <ul class="footer-contact">
-            <li>Contact</li>
-            <li>store@uikit.com</li>
-            <li>Hotline: +1 131 138 138</li>
+        </li>
+        <li class="footer-columns-item">
+          <ul class="columns-info-list">
+            <h5 class="columns-info-title">Contact</h5>
+            <li class="columns-info-item">store@uikit.com</li>
+            <li class="columns-info-item">Hotline: +1 131 138 138</li>
           </ul>
-        </div>
-      </div>
-      <div class="white-line"></div>
-      <p class="copyright">DESIGN BY ICEO.CO - © 2019. ALL RIGHTS RESERVED.</p>
-    </footer>
-  </div>
+        </li>
+      </ul>
+    </div>
+    <div class="white-line"></div>
```

Comment:
Cái này ko cần thiết e nha, đường kẻ đó hoàn toàn có thể CSS từ thẻ p của text cuối